### ISSUE TITLE
Honeyd takes a long time to startup with many nodes

### ISSUE THREAD

**Issue by ghost** [DataSoft/Honeyd#22]
To reproduce:

Make a configuration with many nodes. 500, say. When you try to run honeyd, it will become unresponsive to probes for the first few minutes. (approximately 5 minutes for 500 nodes). After waiting for this duration, then honeyd will respond to probes.

During this time, honeyd is not using any significant amount cpu cycles doing any tasks. So it's likely that whatever is being done here could be removed / done better.

**Comment by DataSoft** [comment]
Problem seems to be that before actually starting, honeyd goes out and makes DNS requests for each of the template names we give it. Very curious.

Here is an sample from strace:

```
socket(PF_INET, SOCK_DGRAM|SOCK_NONBLOCK, IPPROTO_IP) = 13
connect(13, {sa_family=AF_INET, sin_port=htons(53), sin_addr=inet_addr("127.0.0.1")}, 16) = 0
gettimeofday({1344983407, 840977}, NULL) = 0
poll([{fd=13, events=POLLOUT}], 1, 0)   = 1 ([{fd=13, revents=POLLOUT}])
send(13, ",'\1\0\0\1\0\0\0\0\0\0\23CustomNodeProfile-2"..., 37, MSG_NOSIGNAL) = 37
poll([{fd=13, events=POLLIN}], 1, 5000) = 1 ([{fd=13, revents=POLLIN}])
ioctl(13, FIONREAD, [112])              = 0
recvfrom(13, ",'\201\203\0\1\0\0\0\1\0\0\23CustomNodeProfile-2"..., 1024, 0, {sa_family=AF_INET, sin_port=htons(53), sin_addr=inet_addr("127.0.0.1")}, [16]) = 112
close(13)
```

If you pop open wireshark during this time, you will see all the many DNS requests being made for "CustomNodeProfile-2", "CustomNodeProfile-3", etc...

**Comment by DataSoft** [comment]
Line 621 of cconfig.c seems to be the culprit:

```
isipaddr = addr_aton(newtmpl->name, &addr) != -1;
```

This line is doing DNS lookups. Find a way to check if this string is an IP address without using DNS resolving.

**Comment by DataSoft** [comment]
Fixed in 392ae9de509e10adb82752fc93468c0642009233
